### ISSUE TITLE
HAS is not indexed

### ISSUE THREAD

**Issue by JJ** [Raku/Documentable#116]
**Describe the bug**

It's declared here:

```
doc/Language/nativecall.pod6:=head2 Embedding CStructs and CUnions with X<C<HAS>|HAS, declarator>
```

but it does not show up in the index, although it's there:

```
{ category: "Reference", value: "declarator (HAS)", url: "/language/nativecall#index-entry-HAS__declarator-HAS" }
```

**To Reproduce**

Go to docs.raku.org and search HAS

**Expected behavior**

It should show up somewhere, in the "declarator" category or somewhere else

**Additional context**

I don't know if this will apply to all ALL-CAPS entries?

**Comment by antoniogamiz** [comment]
Mm, I see it indexed: `{ category: "declarator", value: "has", url: "/syntax/has" }`. 

Screenshot: 
![1](https://user-images.githubusercontent.com/22995015/87922876-df570900-ca7c-11ea-81ca-70afb86b097f.png)

Do you mean it should appear in capital letters?

**Comment by JJ** [comment]
No, it's ALL CAPS HAS. And it's indexed, only it does not show up in
searches.

**Comment by antoniogamiz** [comment]
Hum, weird, I will take a look.
